app/invocation/cache_requests_card.tsx

```diff
@@ -714,7 +716,17 @@ function looksLikeDigest(actionId: string) {
   return actionId.length === 64;
 }
 
-function toResourceCacheType(cacheType: cache.CacheType) {
+function getCacheType(result: cache.ScoreCard.IResult): resource.CacheType {
+   const cacheType = result.cacheType;
+   // If the cacheType field is not set, try reading data from the older cacheTypeDeprecated field
+   // for scorecard results that were written before we added the new cacheType field
+   if (cacheType == resource.CacheType.UNKNOWN_CACHE_TYPE) {
```

Comment:
nit: IIRC in the protobufjs API `result.cacheType` will be `undefined` if it's unset, so I think this comparison might be relying on the fact that `undefined == 0` in JS (due to `==` instead of `===`).

We generally avoid `==` so it might be slightly clearer to write it as:

```suggestion
   if (!cacheType) {
```

Reply:
Done